commons/css/skolar.css

```diff
@@ -1,20 +1,20 @@
 @font-face {
   font-family: "Skolar";
-  src: url("http://jktauber.com/site_media/static/fonts/SkolarPEWeb-Regular.woff") format("woff");
+  src: url("https://jktauber.com/site_media/static/fonts/SkolarPEWeb-Regular.woff") format("woff");
```

Comment:
Does this work? jktauber.com doesn't have a valid certificate (trusted by browsers).